llvm/lib/CAS/OnDiskCommon.cpp

```diff
@@ -12,6 +12,16 @@
 #include "llvm/Support/Process.h"
 #include <mutex>
 #include <optional>
+#include <thread>
+
+#if __has_include(<sys/file.h>)
+#include <sys/file.h>
```

Comment:
Is this a common unix header or is this macOS only?

Reply:
It's the same on Linux https://man7.org/linux/man-pages/man2/flock.2.html